.github/workflows/codestyle-check.yaml

```diff
@@ -0,0 +1,45 @@
+name: Codestyle check
+on:
+  pull_request:
+    paths:
+      - .github/workflows/codestyle-check.yaml
+      - .clang-format
+      - '**.c'
```

Comment:
No, this approach won't work:
- for 3party files
- for some of our files (e.g. protobuf source breaks after formatting)

I'll provide more details later.